mathics/core/expression.py

```diff
@@ -1416,6 +1418,9 @@ def to_python(self, *args, **kwargs):
             #         keywords=[],
             #         )
             return py_obj
+
+        # Notice that in this case, `to_python` returns a Mathics Expression object,
+        # instead of a builtin native object.
```

Comment:
I expect over time that more iterations over the code will be done and the thing that seem unusual will be addressed or there will be explanations of _why_ they are the way they are.